securityscanutils/securityscan.go

```diff
@@ -233,23 +259,29 @@ func (r *SecurityScanRepo) RunMarkdownScan(ctx context.Context, release *github.
 			imageWithRepo = fmt.Sprintf("%s/%s:%s", r.Opts.ImageRepo, image, version)
 		}
 		fileName := fmt.Sprintf("%s_cve_report.docgen", image)
-		output := path.Join(outputDir, fileName)
+		output := path.Join(trivyScanOutputDir, fileName)
 		_, vulnFound, err := r.trivyScanner.ScanImage(ctx, imageWithRepo, markdownTplFile, output)
 		if err != nil {
-			return eris.Wrapf(err, "error running image scan on image %s", imageWithRepo)
+			if errors.Is(err, UnrecoverableErr) {
+				return eris.Wrapf(err, "error running image scan on image %s", imageWithRepo)
+			}
+			vulnerabilityMd += fmt.Sprintf("# %s\n\n %s", imageWithRepo, err)
```

Comment:
Could you point me to the test that shows this working in both the positive (unrecoverable error is short-circuited) and negative (recoverable error continues)

Reply:
Adding additional tests to `securityscan_test.go` in an upcoming commit for this